common/src/main/java/com/wynntils/core/webapi/ServerListModel.java

```diff
@@ -25,6 +25,10 @@ public static ServerProfile getServer(String worldId) {
         return availableServers.get(worldId);
     }
 
+    public static Map<String, ServerProfile> getAvailableServers() {
```

Comment:
Remove this. This model does not update servers automatically, because it doesn't need to. Instead fetch new info using WebManager where this method is used (in the command)

Reply:
I'll do it when I get home